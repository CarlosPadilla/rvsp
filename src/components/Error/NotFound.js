import React from 'react';
import { Link } from 'react-router-dom';

const NotFound = () => (
  <div>
    <h1>Uups.. eksisid vist Ã¤ra :(</h1>
    <Link to="/">Mine tagasi esilehele</Link>
  </div>
);

export default NotFound;
