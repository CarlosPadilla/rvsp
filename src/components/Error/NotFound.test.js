import React from 'react';
import { shallow } from 'enzyme';
import { MemoryRouter } from 'react-router'
import NotFound from './NotFound';

describe('`Not found` component', () => {

  const app = shallow(<NotFound />)

  it('renders', () => {
    expect(app).toMatchSnapshot()
  })

})
