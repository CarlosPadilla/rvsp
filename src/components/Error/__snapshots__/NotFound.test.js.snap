// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`\`Not found\` component renders 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NotFound />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          Uups.. eksisid vist ära :(
</h1>,
        <Link
          replace={false}
          to="/"
>
          Mine tagasi esilehele
</Link>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Uups.. eksisid vist ära :(",
        },
        "ref": null,
        "rendered": "Uups.. eksisid vist ära :(",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": "Mine tagasi esilehele",
          "replace": false,
          "to": "/",
        },
        "ref": null,
        "rendered": "Mine tagasi esilehele",
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            Uups.. eksisid vist ära :(
</h1>,
          <Link
            replace={false}
            to="/"
>
            Mine tagasi esilehele
</Link>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Uups.. eksisid vist ära :(",
          },
          "ref": null,
          "rendered": "Uups.. eksisid vist ära :(",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": "Mine tagasi esilehele",
            "replace": false,
            "to": "/",
          },
          "ref": null,
          "rendered": "Mine tagasi esilehele",
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
